docs/endpoints/post-optout-status.md

```diff
@@ -0,0 +1,123 @@
+---
+title: POST /optout/status
+description: Checks the opt-out status of raw UID2s.
+hide_table_of_contents: false
+sidebar_position: 03
+---
+
+import Link from '@docusaurus/Link';
+
+# POST /optout/status
+
+Checks the opt-out status of raw UID2s. Given a list of raw UID2s, this endpoint returns the raw UID2s that have opted out, as well as the time that the opt-out took place. For more information, see [User Opt-Out](../getting-started/gs-opt-out.md).
+
+Used by: This endpoint is used by advertisers, data providers, DSPs, and sharers. More generally, this is for participants who have access to raw UID2s but do not have access to the underlying email addresses or phone numbers and want to know opt-out status.
+
+For details, see one of the following guides:
```

Comment:
should this line be reworded to include docs/sharing/sharing-overview for sharers? (the paragraph aboves mentions advertisers/DP/dsp/sharer but only point to integ guide for DSP/DP/advertiser?

Reply:
Great call Sunny. Done. Thx.